aisdc/attacks/likelihood_attack.py

```diff
@@ -529,6 +589,20 @@ def main():
         dest="n_shadow_models",
         help=("The number of shadow models to train (default = %(default)d)"),
     )
+
+    parser.add_argument(
+        "--n-shadow-rows-confidences-min",
+        type=int,
+        action="store",
+        dest="n_shadow_rows_confidences_min",
+        default=2,
```

Comment:
if my interpretation (above) is right, this is way too low for a sensible default value?

Reply:
Yes your interpretation is correct, the default is now set to be 5 instead of 2.